src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: AWS
+description: Configure AWS credentials
+sidebar_label: aws
+id: aws-cloud-credentials
+---
+import Image from "@theme/Image";
+
+
+This tutorial will guide you on how to configure AWS credentials for your Okteto instance. In this specific guide we'll request access to s3 resource but you can provide access to any other AWS resource by using a role with more/different permissions.
```

Comment:
I would be explicit here about the goal of defining AWS credentials. We can explain that deploy/destroy/test commands will get access to your was account using these credentials